src/usb.c

```diff
@@ -15,11 +15,12 @@
     along with libnspire.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <libusb.h>
```

Comment:
Why? System includes normally come before local includes.

Reply:
Windows has the amazing thing where winsocket and winsocket2 are incompatible, but we use winsocket2 elsewhere while libusb uses winsocket internally. Including winsocket2 first overrides winsocket (it defines its header guards), which I include in endianconv.h. Instead, I could place a 3-line `#ifdef _WIN32` above this include if you'd prefer.

Reply:
Sounds like `endianconv.h` should either use `winsock` or nothing to avoid that.

Reply:
Does libusb really use winsock in its public API? That sounds odd...

Reply:
[Yep](https://github.com/libusb/libusb/blob/70c0a51ba3a1daaf8fd51f1457dba86dd389debe/libusb/libusb.h#L63)